libs/js-sdk/src/client-connection.ts

```diff
@@ -8,6 +8,15 @@ export interface BrowserClient {
 
     waitForInitialised(): Promise<boolean>
 
+    /** Subscribe to initialised changes, will call back with initilised boolean value
+     * Recommended to be used in conjuction with updateAudiences()
+     *
+     * @returns unsubscribe function
+     */
+    subscribeToInitialisedChanged(
```

Comment:
I wonder if we should have an `on(event: 'value-changed' | 'initialised-changed', (e: Event) => void)` function instead which is a bit more idiomatic JS. I know I suggested this, but i'm now rethinking it.

Reply:
We discussed, I was wrong. We are going to leave it